app/controllers/SummaryAnswerPageController.scala

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers
+
+import controllers.actions._
+import javax.inject.Inject
+import pages.RegistrationProgress
+import play.api.i18n.{I18nSupport, MessagesApi}
+import play.api.mvc.MessagesControllerComponents
+import uk.gov.hmrc.play.bootstrap.controller.FrontendBaseController
+import utils.CheckYourAnswersHelper
+import utils.countryOptions.CountryOptions
+import views.html.SummaryAnswerPageView
+
+
+class SummaryAnswerPageController @Inject()(
+                                              override val messagesApi: MessagesApi,
+                                              identify: IdentifierAction,
+                                              getData: DataRetrievalAction,
+                                              requireData: DataRequiredAction,
+                                              val controllerComponents: MessagesControllerComponents,
+                                              view: SummaryAnswerPageView,
+                                              countryOptions : CountryOptions,
+                                              registrationProgress: RegistrationProgress
+                                            ) extends FrontendBaseController with I18nSupport {
+
+  private def actions() =
+    identify andThen getData andThen requireData
+
+  def onPageLoad() = actions() {
+    implicit request =>
+
+      val checkYourAnswersHelper = new CheckYourAnswersHelper(countryOptions)(request.userAnswers, isSummary = true)
+      val trustDetails = checkYourAnswersHelper.trustDetails.getOrElse(Nil)
+      val trustees = checkYourAnswersHelper.trustees.getOrElse(Nil)
+      val settlors = checkYourAnswersHelper.settlors.getOrElse(Nil)
+      val individualBeneficiaries = checkYourAnswersHelper.individualBeneficiaries.getOrElse(Nil)
+      val classOfBeneficiaries = checkYourAnswersHelper.classOfBeneficiaries.getOrElse(Nil)
+      val moneyAsset = checkYourAnswersHelper.moneyAsset.getOrElse(Nil)
+      val sections =  Seq() ++ trustDetails ++ settlors ++ trustees ++ individualBeneficiaries ++ classOfBeneficiaries ++ moneyAsset
+
+      val isTaskListComplete = registrationProgress.isTaskListComplete(request.userAnswers)
```

Comment:
@jassalrichy  RegistrationComplete refiner available to check the task list complete, check declaration controller.

Reply:
Done